commands/bridge_configure.go

```diff
@@ -21,26 +33,31 @@ func runBridgeConfigure(cmd *cobra.Command, args []string) error {
 	defer backend.Close()
 	interrupt.RegisterCleaner(backend.Close)
 
```

Comment:
```suggestion

	termState, err := terminal.GetState(int(syscall.Stdin))
	if err != nil {
		return err
	}
	interrupt.RegisterCleaner(func() error {
		return terminal.Restore(int(syscall.Stdin), termState)
	})
```

I did a ctrl+c when the configure was asking for my token (invisible input) and it broke my terminal. This fix the problem.

Reply:
👍  Didn't know about this !